roles/os_hardening/tasks/minimize_access.yml

```diff
@@ -80,3 +80,19 @@
     fstype: proc
     opts: '{{ proc_mnt_options }}'
     state: mounted
+
+- name: Optimize mount options for tmp
+  mount:
+    path: /tmp
+    src: '{{ tmp_mnt_src }}'
+    fstype: '{{ tmp_mnt_filesystem }}'
+    opts: '{{ tmp_mnt_options }}'
+    state: mounted
```

Comment:
This should probably be `present`. When running the task against a running and used server, it will fail with:
```
TASK [devsec.hardening.os_hardening : Optimize mount options for tmp] *******************************************************************************************************************************************fatal: [nrwsp-test-intgr01]: FAILED! => changed=false
  msg: |-
    Error mounting /tmp: umount: /tmp: target is busy.
```

However than a reboot is necessary.

Reply:
there is also a `remount` option (https://docs.ansible.com/ansible/latest/collections/ansible/posix/mount_module.html#parameter-state)

This Option seems to have some caveats. From what I understand, we would need two tasks, one that modifies `/etc/fstab` and one that does the remount.